### DISCUSSION TITLE
Monitor popup keeps showing/doesnt hide in b13

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1034] [General]
The new monitor popup is really more usefull !

I dont know why, but it keeps opening (sometimes with reason) but never disappear ! I tried to disable/enable automatic show but still the same !

I need to click on it to close it on a permanent basis, it is really a small but annoying but :(

Never happened before b13.

**Comment by Henri75** [comment]
And where there is a connection issue it is impossible to close it ! It always reopen

**Reply by apohl79** [reply]
The trigger to open the monitor is if one or more AG plugins are in either "not connected" or "not loaded" state. When you hide it, it stays hidden, until some plugin sends an update and again there is at least one chain not ok. I could relax that logic to ignore any chain that was previously not ok, so it would not trigger the monitor again.

Anyways, you can disable the automatic popup in the monitor menu of the plugin tray icon.

**Comment by Henri75** [comment]
That's how it worked up to to B13.
But now even when everything is green this popup doesnt show hide itself
anymore.
Sometimes it opens with everything green (may be there was an event but
when it opens it is already gone) and the popup stays open with everything
on green.
It never happened before b13

Le lun. 19 sept. 2022 à 10:43, Andreas Pohl ***@***.***> a
écrit :

**Comment by CaBuffalo** [comment]
Yes, I noticed this as well.

Clicking the window makes it goes away.

On Mon, Sep 19, 2022 at 2:39 AM Henri75 ***@***.***> wrote:

> That's how it worked up to to B13.
> But now even when everything is green this popup doesnt show hide itself
> anymore.
> Sometimes it opens with everything green (may be there was an event but
> when it opens it is already gone) and the popup stays open with everything
> on green.
> It never happened before b13
>
> Le lun. 19 sept. 2022 à 10:43, Andreas Pohl ***@***.***> a
> écrit :
>
> > The trigger to open the monitor is if one or more AG plugins are in
> either
> > "not connected" or "not loaded" state. When you hide it, it stays hidden,
> > until some plugin sends an update and again there is at least one chain
> not
> > ok. I could relax that logic to ignore any chain that was previously not
> > ok, so it would not trigger the monitor again.
> >
> > Anyways, you can disable the automatic popup in the monitor menu of the
> > plugin tray icon.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/apohl79/audiogridder/discussions/1034#discussioncomment-3678718
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ANMF6UGFTAGPQV3J3RVGKQLV7ART7ANCNFSM6AAAAAAQPPIPD4
> >
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/1034#discussioncomment-3679071>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BKSBN5GDYQXVSMTVTV7AYDVANCNFSM6AAAAAAQPPIPD4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
I guess the new metrics added in b13 are triggering the monitor popup more often, as there are more changes coming in.

**Comment by CaBuffalo** [comment]
Thanks, Andreas. :)

On Mon, Sep 19, 2022 at 10:23 AM Andreas Pohl ***@***.***>
wrote:

> I guess the new metrics added in b13 are triggering the monitor popup more
> often, as there are more changes coming in.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/1034#discussioncomment-3683278>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GWMWVHLTBUOFIB4KLV7CORPANCNFSM6AAAAAAQPPIPD4>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Kevin Ma
Supported by Gmail.
